src/librustc_mir/interpret/operand.rs

```diff
@@ -568,38 +561,55 @@ impl<'a, 'mir, 'tcx, M: Machine<'a, 'mir, 'tcx>> EvalContext<'a, 'mir, 'tcx, M>
             .collect()
     }
 
-    // Used when Miri runs into a constant, and (indirectly through lazy_const_to_op) by CTFE.
-    fn const_value_to_op(
+    // Used when Miri runs into a constant, and by CTFE.
+    pub fn lazy_const_to_op(
```

Comment:
Could we call this something with `eval`? It's not just a conversion function, after all.

E.g., `eval_lazy_const_to_op`.